apps/web/src/app/(dashboard)/documents/[id]/edit-document.tsx

```diff
@@ -35,7 +35,12 @@ export type EditDocumentFormProps = {
   documentData: DocumentData;
 };
 
-type EditDocumentStep = 'title' | 'signers' | 'fields' | 'subject';
+enum EditDocumentStepEnum {
```

Comment:
Typescript enums are _cursed_ so we avoid them, a const object is a nice middleground that we use elsewhere in the codebase 😄